scripts/update/radarr.sh

```diff
@@ -25,14 +24,8 @@ if [[ $(_radarr_version) != 'mono-v2' ]]; then
 			ret=$(curl -sL "http://127.0.0.1:7878/api/v3/system/status?apiKey=${apikey}")
 			#
 			echo_log_only "Content of ret =\n ${ret}"
```

Comment:
```suggestion
```
As far as I can see `$ret` is not useful anymore if we don't have to use the API